.github/workflows/build-with-bal-test-native.yml

```diff
@@ -29,4 +29,4 @@ jobs:
             native_image_options: ${{ inputs.native_image_options }}
             additional_ubuntu_build_flags: '-x :grpc-native:test -x :grpc-compiler-plugin-tests:test -x :grpc-tooling-tests:test'
             additional_windows_build_flags: '-x :grpc-native:test -x :grpc-compiler-plugin-tests:test -x :grpc-tooling-tests:test'
-            java_tool_options: '-Dfile.encoding=UTF8'
+            java_tool_options: '-Dfile.encoding=UTF8'
```

Comment:
Shall we add a newline here as well. I must have missed this when adding tests

Reply:
Added in [d3cab60](https://github.com/ballerina-platform/module-ballerina-grpc/pull/1122/commits/d3cab606191ea7c4c705a00a36d6439632781502)